openapi/components/schemas/GatewayAccountConfig/BVNK.yaml

```
@@ -26,3 +26,13 @@ allOf:
           - merchantId
           - hawkAuthId
           - hawkAuthKey
+      settings:
+        type: object
+        description: BVNK settings object.
+        properties:
+          payoutCurrency:
+            type: string
+            description: Payout cryptocurrency.
+          payoutNetwork:
+            type: string
+            description: Blockchain network to be used.
```

suggestion: for readability
```suggestion
            description: Blockchain network to use for the payout.
```